debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/SharedClassesMetaDataHelper.java

```diff
@@ -23,14 +23,20 @@
 
 import com.ibm.j9ddr.CorruptDataException;
 import com.ibm.j9ddr.vm29.pointer.generated.J9ShrOffsetPointer;
+import com.ibm.j9ddr.vm29.types.UDATA;
+import java.lang.reflect.*;
```

Comment:
I believe this only imports one class (Method); a single import is preferable to a wildcard in that case.

Reply:
Fixed